README.md

```diff
@@ -36,6 +36,8 @@ It's best if you use a public communication channel whenever possible; however,
 
 ## Success Stories
 
+[Matei David](https://www.cncf.io/spotlights/matei-david/)
```

Comment:
Let's add https://www.cncf.io/spotlights/cncf-maintainer-spotlight-goutham-veeramachaneni/ as well, Goutham is our intern in the past.

Reply:
thanks, didn't know. Have added now